### DISCUSSION TITLE
Manjaro unofficial LXQt iso coming back

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2126] [Media]
https://forum.manjaro.org/t/lxqt-unofficial-minimal-norse-root/90306

Happy to see that!
EDIT: link to the .iso file https://iso.uex.dk/lxqt/manjaro-lxqt-21.2.0-linux515.iso

**Comment by tsujan** [comment]
Haven't tested it but the screenshots show that it's clean, as is usually the case with Manjaro.

**Reply by stefonarch** [reply]
I've installed it in a VM, everything works perfectly. Looks like qps is not available?

**Comment by edwardp3** [comment]
I burned the image to a DVD and tried it out, it looks nice.

I liked the Openbox theme it used in the screenshots and found one that is similar: https://store.kde.org/p/1416095/ (Nord Openbox)

**Reply by tsujan** [reply]
From their screenshot, it seems they may have used Kvantum for the Qt widget style, with a dark theme that matches their GTK theme.

**Comment by stefonarch** [comment]
I've found this one  with square buttons and apple colors once, but cannot retrieve it anymore. 
My favourite is _vent_ from [here](https://github.com/addy-dclxvi/openbox-theme-collections)
![schermata-11-20-20-52](https://user-images.githubusercontent.com/10681413/142739269-901bdc02-85b9-4466-958a-7f9fa00b8793.png).

**Reply by edwardp3** [reply]
Just installed Vent. Nice!

**Comment by stefonarch** [comment]
Another spin based on Manjaro with LXQt:
https://spins.tuxfamily.org/spins.html
https://osdn.net/projects/spins-by-kilz/news/28502

**Comment by nlsmart** [comment]
Hi,

On my side I build the ISO images with the Manjaro tools, x86 and arm available.

https://forum.manjaro.org/t/build-iso-image-lxqt/90181

https://forum.manjaro.org/t/odroid-n2-manjaro-arm-21-10-lxqt-1-0-0/90323

**Comment by stefonarch** [comment]
I've just noticed that now two "community" editions are now  available:
https://manjaro.org/download/

![schermata-03-21-07-29-01](https://user-images.githubusercontent.com/10681413/159214370-6ce231b7-be1e-4b7d-868a-55bd201a6d82.png)

**Comment by tsujan** [comment]
At last, I found some time to try Manjaro LXQt ISO in VirtualBox. It was impressive! Kudos to the Manjaro team.
